plugins/caches.js

```diff
@@ -291,6 +291,28 @@ exports.plugin = {
                             id: SCHEMA_SCOPE_ID,
                             cacheName: SCHEMA_CACHE_NAME
                         })
+                    },
+                    ext: {
+                        onPreAuth: {
+                            method: (request, h) => {
+                                const contentLength = request.headers['content-length'];
+                                const expectHeader = request.headers.expect;
+
+                                if (expectHeader && expectHeader.toLowerCase() === '100-continue') {
+                                    if (contentLength && parseInt(contentLength, 10) > options.maxByteSize) {
+                                        request.log(
+                                            'Payload content length greater than maximum allowed',
+                                            options.maxByteSize
```

Comment:
This log seems to only output the server's max byte size. Is it necessary?

Reply:
Thank you for your comment.
This log entry includes a variable. I wanted to write it in one line, but the linter gave an error, so I had to split it.
I want to log when the `maxByteSize` limit is reached. The number is just for reference.